sdk/storage/storage-queue/review/storage-queue.api.md

```diff
@@ -187,20 +194,24 @@ export const logger: import("@azure/logger").AzureLogger;
 
 // @public
 export interface Logging {
-    deleteProperty: boolean;
+    delete: boolean;
     read: boolean;
-    // (undocumented)
     retentionPolicy: RetentionPolicy;
     version: string;
     write: boolean;
 }
 
 // @public
-export interface MessageIdDeleteHeaders {
+export interface MessageIdDeleteExceptionHeaders {
     clientRequestId?: string;
-    date?: Date;
     // (undocumented)
     errorCode?: string;
+}
+
+// @public
+export interface MessageIdDeleteHeaders {
```

Comment:
It looks like we are removing the errorCode property from the `MessageIdDeleteHeaders` compared with the previous version, is that correct? That would be a breaking change, was it removed from the SWAGGER?

Reply:
It was not "removed" from the swagger. In there are 2 responses, say "200" and "default", in v6 generator we have 2 headers. (One is for 200 & One for default exception). In V4 we used to have only one header. So, the error code is included in it by default. To avoid breaking changes, I have included it by swagger transform.